missions/ULM_missions.txt

```diff
@@ -0,0 +1,335 @@
+		required_missions = { ULM_imperial_guarantee }
+		position = 4
+		
+		trigger = {
+			army_size_percentage = 0.8
+			any_country = {
+				rival = ROOT
+				OR = {
+					primary_culture = swiss
+					primary_culture = swabian
+				}
+			}
+		}
+
+		effect = {
+			country_event = ULM_zael_event.1
+		}
+	}
+	
+	ULM_free_city_guard = {
+		icon = mission_lanshknecht_soldier.png
+		required_missions = { ULM_swabians_and_the_swiss ULM_defensive_overhaul }
+		position = 6
+		
+		trigger = {
+			army_tradition = 60
+			army_professionalism = 20
+			OR = { 
+				any_ally = {
+					primary_culture = swabian
+					has_opinion = {
+						who = ROOT
+						value = 150
+					}
+					army_professionalism = 40
+				}
+				any_ally = {
+					primary_culture = swiss
+					has_opinion = {
+						who = ROOT
+						value = 150
+					}
+					army_professionalism = 40
+				}
+				any_country = {
+					NOT = {
+						OR = {
+							primary_culture = swiss
+							primary_cultre = swabian
+						}
+					}
+				}
+			}
+		}
+		
+		effect = {
+			add_mil_power = 300
+			add_army_professionalism = 10
+			country_event = ULM_zael_event.2
+		}
+	}
+}
+
+ULM_imperial_misc_missions = {
+	slot = 2
+	generic = no
+	ai = yes
+	potential = {
+		tag = ULM
+	}
+	has_country_shield = yes
+	
+	ULM_imperial_bankers = {
+		icon = mission_high_income.png
+		required_missions = { ULM_imperial_guarantee }
+		position = 3
+		
+		trigger = {
+			diplomatic_reputation = 4
+			mercantilism = 15
+			OR = {
+				ROOT = {
+					guaranteed_by = emperor
+					NOT = { inflation = 4 }
+					monthly_income = 4
+				}
+				ROOT = {
+					NOT = { inflation = 2 }
+					monthly_income = 8
+				}
+			}
+		}
+		
+		effect = {
+			country_event = ULM_zael_event.3
+			add_mercantilism = 10
+			hidden_effect = {
+				limit = { guaranteed_by = emperor }
+				add_imperial_influence = 20
+			}
+		}
+	}
+	
+	ULM_defensive_overhaul = {
+		icon = mission_conquer_50_development.png
+		required_missions = { ULM_swabians_and_the_swiss ULM_establish_milf }
+		position = 5
+		
+		trigger = {
+			OR = { 
+				any_ally = {
+					primary_culture = swabian
+					has_opinion = {
+						who = ROOT
+						value = 150
+					}
+					army_tradition = 70
+				}
+				any_ally = {
+					primary_culture = swiss
+					has_opinion = {
+						who = ROOT
+						value = 150
+					}
+					army_tradition = 70
+				}
+				any_country = {
+					NOT = {
+						OR = {
+							primary_culture = swiss
+							primary_cultre = swabian
+						}
+					}
+				}
+			}
+            1872 = {
+				type = all
+				OR = {
+					has_building = fort_15th
+					has_building = fort_16th
+					has_building = fort_17th
+					has_building = fort_18th
+				}
+            }
+			treasury = 1000
+		}
+		
+		effect = {
+			treasury = -600
+			1872 = {
+				add_permanent_province_modifier = {
+					name = ULM_defensive_overhaul
+					duration = -1
+				}
+			}
+			custom_tooltip = emp_mlo_fortify_alps_effects_tooltip
+			hidden_effect = {
+				every_owned_province = {
```

Comment:
Should replace `every_owned_province = {` with just `1872 = {` and remove the limit.